### PR TITLE
Change Labeling colors

### LINKED ISSUE #212: label.py should use colors that are more different to each other
Green and yellow are very similar.

### PR DESCRIPTION
**Pull request by Shashi456**
Fixes #212.

### CODE REVIEW

**Comment by Shashi456** [comment]
@marco-c should i make a seperate pr to reflect the changes in readme.md? we would need to change the images

**Comment by sagarvijaygupta** [comment]
@Shashi456 You can wish you can try [`Adobe Color Wheel`](https://color.adobe.com/). I am pretty bad at colors even after using this.

**Comment by Shashi456** [comment]
@sagarvijaygupta  i will check it out thank you

**Comment by marco-c** [comment]
> @marco-c should i make a seperate pr to reflect the changes in readme.md? we would need to change the images

No, we should change them at the same time, otherwise we leave the repository in an inconsistent state.

**Comment by Shashi456** [comment]
image

**Comment by Shashi456** [comment]
image

File: README.md [hunk 1]
```diff
@@ -86,9 +86,9 @@ For the unsupervised training, we are using a related problem for which we alrea
 
     ```
     -network                  To select which network architecture to use
-                                                       
+
```
Comment (marco-c) [inline README.md]: These whitespace changes are unrelated, could you remove them from the patch?

File: label.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 box_to_change = {}
 all_boxes = {}
 key_map = {'Escape': 27, 'r': 114, 'Enter': 13, 'Space': 32, 'y': 121, 'left_a': 97, 'right_d': 100}
-COLOR_N = (0, 255, 0)  # GREEN
+COLOR_N = (128, 0, 128)  # PURPLE
```
Comment (marco-c) [inline label.py]: You need to update the help text at the bottom of the file too.

File: README.md (hunk 1 above)
Comment (Shashi456) [reply README.md]: i have no idea why this keeps happening. I didn't touch this part of the file at all. i tried to revise this but this still keep popping up

Comment (marco-c) [reply README.md]: It's probably your text editor doing this automatically. It's OK, let's ignore it.
